2016/proposals/vora-meet.md

```diff
@@ -105,22 +105,23 @@ class WorkshopSurvey(models.Model):
 
 The *results* field for a WorkshopSurvey object will store JSON in the following format 
 ```python
-[	
-	{
-		'id': 		question_id,
-		'answer':	[
-						{
-							'id': 		answer_id
-							'count':	answer_count['answer_id']	 	# answer_count shall be user-defined dictionary as described above
-						},
-					]
-	},
-]
+{	
+	'index' :	
+		{
+			'id': 		question_id,
+			'answer':	[
+							{
+								'id': 		answer_id
+								'count':	answer_count['answer_id']	 	 #answer_count shall be user-defined dictionary as described above
+							},
+						]
+		},
+}
```

Comment:
Minor thing: it's better to use space-based indentation (with the indent of 4 spaces).